### ISSUE TITLE
When a non-existing file is given, the error handling is not executed 

### ISSUE THREAD

**Issue by plaenen** [tobius/imagecolors#3]
Hi, 

I think I found a small error in your code.
The way the fileExists is handled is not working when the file doesn't exist. 

I suggest the following change in the main.js for the following reasons: 
1. the sync call is removed, better not to block IO
2. when a file isn't found a nice error is given back

``` javascript
    convertPathToImage: function(path, callback){
        fs.lstat(path, function(err, stats) {
            if (err){
                return callback(err,null); 
            };
            if (path.match(/^htt/) || stats.isFile()){
                return callback(undefined, im(path));
            }
            else {
                callback(new Error('Unable to quantify image path'), null);    
            }
        });
    },
```

**Comment by carsy** [comment]
:+1: same here

**Comment by tobius** [comment]
fixed in 0.11.0
